library/core/src/slice/ascii.rs

```diff
@@ -67,10 +67,15 @@ impl [u8] {
     ///
     /// [`to_ascii_uppercase`]: #method.to_ascii_uppercase
     #[stable(feature = "ascii_methods_on_intrinsics", since = "1.23.0")]
+    #[rustc_const_unstable(feature = "const_make_ascii", issue = "130698")]
     #[inline]
-    pub fn make_ascii_uppercase(&mut self) {
-        for byte in self {
+    pub const fn make_ascii_uppercase(&mut self) {
+        // FIXME(const-hack): We would like to simply iterate using `for` loops but this isn't currently allowed in constant expressions.
+        let mut i = 0x0;
```

Comment:
Why hex literals?

Reply:
Sorry, I overlooked those. It's a habit.